runtime-builder/tests/integration/test.sh

```diff
@@ -34,9 +34,12 @@ if [[ "${use_rb}" = "False" || "${rb_root}" != file://* ]]; then
     exit 1
 fi
 
+: ${STAGING_BUILDER_IMAGE?"Staging builder image path not set."}
```

Comment:
Is this script used by the integration test framework as well?  If not, can we passed in only the tag value as a param instead of setting the environment variable $STAGING_BUILDER_IMAGE, and then set ${STAGING_BUILDER_IMAGE} within this script instead ...

STAGING_BUILDER_IMAGE=gcr.io/gcp-runtimes/go1-builder:${TAG}
envsub '${STAGING_BUILDER_IMAGE}' < test.yaml.in > test.yaml